src/language-js/printer-estree.js

```diff
@@ -4111,7 +4111,7 @@ function printMemberChain(path, options, print) {
   // letter, just a sequence of _$ or this. The rationale is that they are
```

Comment:
While you’re here, could you update this comment (remove the “or this” part and add something about your new changes)?